notebooks/v1/ch-algorithms/shor.ipynb

```diff
@@ -131,7 +131,7 @@
    "source": [
     "## 2. The Solution\n",
     "\n",
-    "Shor’s solution was to use [quantum phase estimation](./quantum-phase-estimation.html) on the unitary operator:\n",
+    "Shor’s solution was to use [quantum phase estimation](./quantum-phase-estimation.ipynb) on the unitary operator:\n",
```

Comment:
this will also return a 404 if you try it:

https://learn.qiskit.org/course/ch-algorithms/quantum-phase-estimation.ipynb

the correct URL path should just be

```suggestion
    "Shor’s solution was to use [quantum phase estimation](./quantum-phase-estimation) on the unitary operator:\n",
```